internal/services/disks/disk_pool_iscsi_target_lun_resource_test.go

```diff
@@ -124,6 +124,8 @@ func (r DisksPoolIscsiTargetLunResource) Exists(ctx context.Context, clients *cl
 }
 
 func (r DisksPoolIscsiTargetLunResource) Destroy(ctx context.Context, clients *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	ctx, cancel := context.WithTimeout(ctx, 60*time.Minute)
```

Comment:
Without creating new context with timeout, the default timeout of 15 minute of Azure API will be used.